src/ReachSets/DiscretePost/ConcreteDiscretePost.jl

```diff
@@ -123,9 +123,10 @@ function post(𝒫::ConcreteDiscretePost,
             if isempty(R⋂G)
                 continue
             end
+
             # apply assignment
-            # TODO converting to HPolytope; this should be handled automatically
-            A⌜R⋂G⌟ = linear_map(assignment, R⋂G, output_type=HPolytope)
+            A⌜R⋂G⌟ = convert(HPolytope, linear_map(assignment, R⋂G))
```

Comment:
We should either keep the TODO as a reminder or get rid of the conversion at all (if it works).

Reply:
Ok, let's for now keep the TODO, then my change is just an equivalent of the previous one